spec/cspec/c/kernel.mk

```diff
@@ -8,29 +8,39 @@
 # It allows building the C kernel in locations other than the default one used by l4v,
 # and assumes that KERNEL_BUILD_ROOT has already been set to specify the build location.
 
+ifndef KERNEL_BUILD_ROOT
+  $(error KERNEL_BUILD_ROOT is not set)
+endif
+
 ifndef L4V_REPO_PATH
   L4V_REPO_PATH := $(realpath $(dir $(lastword ${MAKEFILE_LIST}))../../..)
 endif
 
-ifndef SOURCE_ROOT
-  SOURCE_ROOT := $(realpath ${L4V_REPO_PATH}/../seL4)
+ifndef KERNEL_SOURCE_ROOT
+  KERNEL_SOURCE_ROOT := $(realpath ${L4V_REPO_PATH}/../seL4)
```

Comment:
Any specific reason for the rename? (It's going to conflicts with other build system PRs I have in the pipeline)

Reply:
No strong reason. I think it was just inspired by `KERNEL_BUILD_ROOT`. I'll change it back.